app.rb

```diff
@@ -1,163 +1,20 @@
-require_relative './classroom'
+require_relative './book'
+require_relative './person'
 require_relative './student'
 require_relative './teacher'
-require_relative './book'
 require_relative './rental'
+require_relative './modules/show_books'
+require_relative './modules/show_person'
```

Comment:
- [ ] I think you have the wrong path here since your module called show people but you are calling it as show person so I think you should update the name path here so please fix it 🙏🏻💯🔥

![image](https://user-images.githubusercontent.com/58553711/211436585-949124c0-d00f-40e4-a9cc-7c969d71272d.png)